multipaz/src/commonMain/kotlin/org/multipaz/mdoc/request/DeviceRequestGenerator.kt

```diff
@@ -67,9 +71,33 @@ class DeviceRequestGenerator(
         requestInfo: Map<String, ByteArray>?,
         readerKey: EcPrivateKey?,
         signatureAlgorithm: Algorithm,
-        readerKeyCertificateChain: X509CertChain?
+        readerKeyCertificateChain: X509CertChain?,
+        zkSystemSpecs: List<ZkSystemSpec>? = null
```

Comment:
Maybe just `List<ZkSystemSpec> = emptyList()`, generally it's best to avoid nullable types if you can.